app/views/better_homepage/_tag_cluster.html.erb

```
@@ -3,14 +3,16 @@
   <ul class="c-list c-list--horiz">
     <% omit ||= [] %>
     <% tag.featured_content(omit).each do |feature| %>
-      <div class="o-media o-media--horizontal o-media--sm">
-        <div class="o-media__figure o-media__figure--four-by-three">
-          <img class="o-media__img" src="<%= begin feature.asset.asset.thumb.url rescue '/static/images/fallback-img-square.png' end %>">
+      <% if feature.present? %>
```

Ideally, it's better to fail early so instead of wrapping this all in an `if` block, how about something like:
```
<% tag.featured_content(omit).each do |feature| %>
   <% next if feature.nil? %>
   <div class="o-media....>
```

---

Ah! That's a lot better. Thanks for the review @nakedsushi!